consolidates git provider options (#6691)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/git/managed-repository.md

---
title: "Connect with managed repository"
id: "managed-repository"
description: "Learn how to set up a project with a managed repository."
pagination_next: "docs/cloud/git/import-a-project-by-git-url"
pagination_prev: "docs/cloud/git/git-configuration-in-dbt-cloud"
---

If you don't already have a Git repository for your dbt project, you can let dbt Cloud manage a repository for you. Managed repositories are a great way to trial dbt without needing to create a new repository.

:::info

> We do not recommend using a managed repository in a production environment. You will not be able to use git features like pull requests which are part of our recommended version control best practices.

Reviewer: ```suggestion
dbt Labs recommends against using a managed repository in a production environment. You can't use Git features like pull requests, which are part of our recommended version control best practices.
```

:::

To set up a project with a managed repository:

> 1. From your A**ccount settings** in dbt Cloud, select the project you want to set up with a managed repository. If the project already has a repository set up, you need to edit the repository settings and disconnect the existing repository.

Reviewer: ```suggestion
1. From your **Account settings** in dbt Cloud, select the project you want to set up with a managed repository. If the project already has a repository set up, you need to edit the repository settings and disconnect the existing repository.
```

Author: yikes good catch!

2. Click **Edit** for the project.
3. Under Repository, click **Configure repository**.
4. Select **Managed**.
5. Enter a name for the repository. For example, "analytics" or "dbt-models."
6. Click **Create**.
   <Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/managed-repo.png" title="Adding a managed repository"/>

> - dbt Cloud will host and manage this repository for you.

Reviewer: Removing some future tense
```suggestion
- dbt Cloud hosts and manages this repository for you.
```
